### PR TITLE
Master-KiranW

### PR DESCRIPTION
**Pull request by Kwoochit**
Kiran Woochit pull request

### CODE REVIEW

File: public/stylesheets/main.css [hunk 1]
```diff
@@ -37,3 +38,77 @@ h3 {
 }
 
 /* TODO - Add your own styles here */
+
+/* Headings */
+
+#page-title {
+font-style: italic;
```
Comment (dmgarland) [inline public/stylesheets/main.css]: Try and get in the habit of indenting these to make it easier for others to read.

File: public/stylesheets/main.css [hunk 2]
```diff
@@ -37,3 +38,77 @@ h3 {
...
+font-size: 5em;
+text-align: center;
+}
+
+div h2 {
+  text-align: center;
+  font-size: 3em;
+  font-weight: bold;
+  margin-bottom: 10px;
```
Comment (dmgarland) [inline public/stylesheets/main.css]: Be consistent in your CSS - if you're using ems for fonts might as well make your margins ems too...

**Comment by dmgarland** [comment]
Good start Kiran, I'd like to see more of your JS / Ruby skills by going further on this exercise. Any questions please let me know.

**Comment by Kwoochit** [comment]
Thanks for the feedback Dan, I'll take everything on board and make some adjustments. 
I just wanted to ask, how much time do I have left to finalise everything for the groundwork? I remember Kin mentioning that the groundwork should be completed for the end of October, but I just wanted to see if this still stands. Look forward to hearing from you soon. 
Kind regards,Kiran

Date: Thu, 29 Oct 2015 04:49:51 -0700

**Pull request closed without merging**